ethclient/simulated/backend_test.go

```diff
@@ -228,15 +228,13 @@ func TestForkResendTx(t *testing.T) {
 	if err := sim.Fork(parent.Hash()); err != nil {
 		t.Errorf("forking: %v", err)
 	}
-
 	// 5.
 	sim.Commit()
 	if err := client.SendTransaction(ctx, tx); err != nil {
```

Comment:
It seems `SendTransaction` can be omitted if the `tx` is expected to be picked up automatically?

Reply:
Good point, thanks